draft-ietf-rtcweb-jsep.xml

```diff
@@ -827,8 +827,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         remote endpoint will only send a single encoding per m=
         section. In addition, when the JSEP endpoint is the answerer,
         the permitted encodings for the RTPSender must be consistent
-        with the offer, but this information is currently not surfaced
-        through any API. This means that established simulcast streams
+        with the offer. This means that established simulcast streams
```

Comment:
The surrounding text doesn't really make sense when this clause is removed, e.g. the later points about needing SDP inspection. 

If we are going to say that we can support a simulcast offer, we need to describe how that would work.